lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -1123,8 +1123,10 @@ int mme_app_handle_create_sess_resp(
             "Bad pdn id (%d) for bearer\n", pdn_cx_id);
         continue;
       }
-      ue_context_p->pdn_contexts[pdn_cx_id]->s_gw_teid_s11_s4 =
-          create_sess_resp_pP->s11_sgw_fteid.teid;
+      if (ue_context_p->pdn_contexts[pdn_cx_id]) {
```

Comment:
@ardzoht just curious why would this be NULL ?

Reply:
As to why this would be NULL - could one of the many generators of messages on the ZMQ bus is perhaps allocating a MessageDef e.g. [Here](https://github.com/magma/magma/blob/327600a/lte/gateway/c/oai/lib/itti/intertask_interface.c#L267) but then not filling the message data (leaving it zero-initialized as seen) - which then would be interpreted as a NULL ptr (well it would be a null ptr)?

Reply:
I realize my comment is off - the ZMQ message is not at play here.

Reply:
If the pdn context is not there, either there should de a delete session request processed or default bearer should be deactivated, while there is still a create session request pending. @ardzoht what is the context here?

Reply:
There's a delete session request procedure for theUE on the logs I've looked at related to the coredumps, so I think would be the former case you mention @ulaskozat